front/app/containers/Admin/projectFolders/containers/settings/ProjectFolderForm/index.tsx

```diff
@@ -401,25 +407,27 @@ const ProjectFolderForm = ({ mode, projectFolderId }: Props) => {
                 imageId: folderCardImageToRemove.id,
               });
 
-            const filesToAddPromises = projectFolderFiles
-              .filter((file) => !file.remote)
-              .map((file) =>
-                addProjectFolderFile({
-                  projectFolderId,
-                  file: file.base64,
-                  name: file.name,
-                })
-              );
-            const filesToRemovePromises = projectFolderFilesToRemove.map((id) =>
-              deleteProjectFolderFile({ projectFolderId, fileId: id })
+            const initialFileOrdering = projectFolderFilesRemote?.data.reduce(
+              (acc, file) => {
+                if (file.id) {
+                  acc[file.id] = file.attributes.ordering;
+                }
+                return acc;
+              },
+              {}
             );
 
+            syncProjectFolderFiles({
```

Comment:
You are not awaiting this call. I would do what I also suggested in the other PR: something like

```
const folderFilePromises = syncProjectFolderFiles({
```

and await those promises in the `Promise.all` call below like how it was set up before